gcloud-java-contrib/gcloud-java-nio/src/main/java/com/google/cloud/storage/contrib/nio/CloudStorageFileSystem.java

```diff
@@ -145,10 +145,12 @@ public CloudStoragePath getPath(String first, String... more) {
    * Does nothing.
    */
   @Override
-  public void close() {}
+  public void close() throws IOException {
```

Comment:
Just wondering: how much would it take to actually close all channels rather that postponing this?